ext/opcache/jit/zend_jit_ir.c

```diff
@@ -16811,6 +16811,9 @@ static int zend_jit_trace_end_loop(zend_jit_ctx *jit, int loop_ref, const void *
 
 static int zend_jit_trace_return(zend_jit_ctx *jit, bool original_handler, const zend_op *opline)
 {
+	if (!original_handler && jit->reuse_ip) {
```

Comment:
I think the check for ``original_handler`` is useless, but the check for ``jit->reuse_ip`` may be not enough.

Reply:
Wanted to leave minimal impact on codepaths, but maybe it's useless. I don't understand the code well enough.